default.json

```diff
@@ -22,6 +22,7 @@
         "logging_enabled": true,
         "safe_logging": false,
         "private_build": true,
+        "mls_support_enabled": true,
```

Comment:
don't we need to add this field to other configs/build types?

Reply:
No, because I've defined it as `mls_support_enabled: false` at the top level, so all other build flavour will have it disabled.